# msg/ProcessingState.msg

std_msgs/Float64MultiArray graph_data
std_msgs/Float64MultiArray graph_times

std_msgs/Float64MultiArray peak_times
rr_tracking/StableRange[] stable_ranges

std_msgs/Bool recording

std_msgs/Float32 rr_inst
std_msgs/Float32 rr_avg
